src/test/java/teammates/test/cases/logic/StudentsLogicTest.java

```diff
@@ -69,7 +69,7 @@ public void testAll() throws Exception {
         testAdjustFeedbackResponseForEnrollments();
 
         testValidateSections();
-        testupdateStudentCascadeWithoutDocument();
+        testupdateStudentCascade();
```

Comment:
Camel case naming -> `testUpdateStudentCascade`

Reply:
i didn't notice this since it was already not camel case . bt i changed it now.